src/hooks/useFormElementOptions.ts

```diff
@@ -41,19 +45,44 @@ export default function useFormElementOptions<T>({
     })
   }, [conditionallyShownOptions, element.options, onFilter])
 
+  //
+  const withInjectedOptions = React.useMemo(() => {
+    return shownOptions.reduce<FormTypes.ChoiceElementOption[]>(
+      (memo, option) => {
+        //No replaceable stuff
+        if (
+          !formElementsService.ElementWYSIWYGRegex.test(option.label) &&
+          !formElementsService.ElementWYSIWYGRegex.test(option.value)
+        ) {
+          memo.push(option)
+          return memo
+        }
+        const newOptions = processInjectableOption({
+          option,
+          submission: formSubmissionModel,
+          formElements: elements,
+        })
```

Comment:
Ideally we use the `useTaskContext()` and `useAuth()` hooks in the component and pass those in instead of passing `undefined` for task properties and generating the profile every time.

Reply:
Ah yep, just copied what Dave had for those params